packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -1,24 +1,74 @@
 import * as React from 'react';
-import { useForceUpdate } from '@fluentui/react-utilities';
+import { useEventCallback, useForceUpdate } from '@fluentui/react-utilities';
 import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
-import { Toaster } from './vanilla/toaster';
-import { Toast, ToastPosition, ToasterOptions } from './types';
+import { createToaster } from './vanilla';
```

Comment:
```suggestion
import { createToaster } from './createToaster';
```

?

Reply:
well it still is vanilla actually, along with the dispatchers